geomesa-accumulo/geomesa-accumulo-spark/src/main/scala/org/locationtech/geomesa/spark/accumulo/AccumuloSpatialRDDProvider.scala

```diff
@@ -55,36 +44,12 @@ class AccumuloSpatialRDDProvider extends SpatialRDDProvider with LazyLogging {
 
     lazy val transform = query.getHints.getTransformSchema
 
-    def queryPlanToRDD(sft: SimpleFeatureType, qp: AccumuloQueryPlan, conf: Configuration): RDD[SimpleFeature] = {
+    def queryPlanToRDD(sft: SimpleFeatureType, qp: AccumuloQueryPlan): RDD[SimpleFeature] = {
       if (ds == null || sft == null || qp.isInstanceOf[EmptyPlan]) {
         sc.emptyRDD[SimpleFeature]
       } else {
-        // note: we've ensured there is only one table per query plan, below
-        InputConfigurator.setInputTableName(classOf[AccumuloInputFormat], conf, qp.tables.head)
```

Comment:
All this is now in the GeoMesaAccumuloInputFormat, right?

Reply:
right, i created a Job instance to temporarily pass through to those methods and then retrieve the config from it. seemed to work ok...

Reply:
Sweet.  I assume you tested on all 8 million combinations of MapReduce, HBase, Accumulo, Linux, public cloud provider, and day of the week.  Works for me!